public/app/features/alerting/unified/components/mute-timings/MuteTimingForm.tsx

```diff
@@ -30,47 +30,49 @@ interface Props {
   muteTiming?: MuteTimeInterval;
   showError?: boolean;
   provenance?: string;
+  loading?: boolean;
 }
 
 const useDefaultValues = (muteTiming?: MuteTimeInterval): MuteTimingFields => {
-  return useMemo(() => {
-    const defaultValues = {
-      name: '',
-      time_intervals: [defaultTimeInterval],
-    };
+  const defaultValues = {
+    name: '',
+    time_intervals: [defaultTimeInterval],
+  };
 
-    if (!muteTiming) {
-      return defaultValues;
-    }
-
-    const intervals = muteTiming.time_intervals.map((interval) => ({
-      times: interval.times ?? defaultTimeInterval.times,
-      weekdays: interval?.weekdays?.join(', ') ?? defaultTimeInterval.weekdays,
-      days_of_month: interval?.days_of_month?.join(', ') ?? defaultTimeInterval.days_of_month,
-      months: interval?.months?.join(', ') ?? defaultTimeInterval.months,
-      years: interval?.years?.join(', ') ?? defaultTimeInterval.years,
-    }));
-
-    return {
-      name: muteTiming.name,
-      time_intervals: intervals,
-    };
-  }, [muteTiming]);
+  if (!muteTiming) {
+    return defaultValues;
+  }
+
+  const intervals = muteTiming.time_intervals.map((interval) => ({
+    times: interval.times ?? defaultTimeInterval.times,
+    weekdays: interval.weekdays?.join(', ') ?? defaultTimeInterval.weekdays,
+    days_of_month: interval.days_of_month?.join(', ') ?? defaultTimeInterval.days_of_month,
+    months: interval.months?.join(', ') ?? defaultTimeInterval.months,
+    years: interval.years?.join(', ') ?? defaultTimeInterval.years,
+    location: interval.location ?? defaultTimeInterval.location,
+  }));
+
+  return {
+    name: muteTiming.name,
+    time_intervals: intervals,
+  };
 };
 
 const defaultPageNav: Partial<NavModelItem> = {
   icon: 'sitemap',
 };
 
-const MuteTimingForm = ({ muteTiming, showError, provenance }: Props) => {
+const MuteTimingForm = ({ muteTiming, showError, loading, provenance }: Props) => {
```

Comment:
`loading` is now being passed in to the form because for some reason the form contains the `AlertingPageWrapper`. We were showing multiple loaders before and one of them was even rendered entirely outside of the application.